### PR TITLE
fix(tooltip): Fix tooltip alignment in toolbar

### PR DESCRIPTION
**Pull request by mindreeper2420**
Fix tooltip alignment in the Codebases toolbar; add text to provide more spacing and meet future design specs.

fixes: https://github.com/openshiftio/openshift.io/issues/1179

### CODE REVIEW

**Comment by fabric8cd** [comment]
@mindreeper2420 fabric8/fabric8-ui:SNAPSHOT-PR-2245-1 fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2245-fabric8-ui.badger.fabric8.io

File: src/app/space/create/codebases/codebases-toolbar/codebases-toolbar.component.html [hunk 1]
```diff
@@ -5,9 +5,12 @@
       (onFilterChange)="filterChange($event)"
       (onSortChange)="sortChange($event)">
     <ng-template #addCodebaseTemplate>
-      <div class="with-cursor-pointer" placement="left" tooltip="Add A Codebase"
+      <div class="with-cursor-pointer" placement="left" tooltip="Add a Codebase"
            [routerLink]="[{ outlets: { action: 'add-codebase' } }]">
-        <i class="pficon pficon-add-circle-o margin-top-4"></i>
+        <span class="add-codebase-tooltip">
+          <i class="pficon pficon-add-circle-o margin-top-4"></i>
+          <span class="add-codebase-tooltip-text">Add Codebase</span>
```
Comment (dlabrecq) [inline src/app/space/create/codebases/codebases-toolbar/codebases-toolbar.component.html]: add-codebase-tooltip-text appears to be unused?

File: src/app/space/create/codebases/codebases-toolbar/codebases-toolbar.component.ts [hunk 2]
```diff
@@ -73,7 +63,8 @@ export class CodebasesToolbarComponent implements OnInit {
       appliedFilters: [],
       resultsCount: this.resultsCount,
       selectedCount: 0,
-      totalCount: 0
+      totalCount: 0,
+      tooltipPlacement: 'left'
```
Comment (dlabrecq) [inline src/app/space/create/codebases/codebases-toolbar/codebases-toolbar.component.ts]: This tooltipPlacement is for the filter only. Currently, it appears truncated while positioned on the left.

screen shot 2017-11-20 at 3 37 18 pm

File: src/app/space/create/codebases/codebases-toolbar/codebases-toolbar.component.html [hunk 3]
```diff
@@ -5,9 +5,12 @@
       (onFilterChange)="filterChange($event)"
       (onSortChange)="sortChange($event)">
     <ng-template #addCodebaseTemplate>
-      <div class="with-cursor-pointer" placement="left" tooltip="Add A Codebase"
+      <div class="with-cursor-pointer" placement="left" tooltip="Add a Codebase"
```
Comment (dlabrecq) [inline src/app/space/create/codebases/codebases-toolbar/codebases-toolbar.component.html]: Might want to try the bottom? When the window gets small, the tooltip is pushed off the page.

screen shot 2017-11-20 at 3 55 40 pm

Comment (mindreeper2420) [reply src/app/space/create/codebases/codebases-toolbar/codebases-toolbar.component.html]: I was unable to fix this, when it is on the left. When set to 'bottom', it causes this:
screen shot 2017-11-20 at 5 35 21 pm
screen shot 2017-11-20 at 5 35 28 pm

When set to "top", it causes a crazy jumping screen, as well as this:
screen shot 2017-11-20 at 5 36 31 pm

I'll try some CSS positioning changes, but the piece that is in place isn't super flexible.

File: src/app/space/create/codebases/codebases-toolbar/codebases-toolbar.component.html (hunk 1 above)
Comment (mindreeper2420) [reply src/app/space/create/codebases/codebases-toolbar/codebases-toolbar.component.html]: removed

**Comment by fabric8cd** [comment]
@mindreeper2420 fabric8/fabric8-ui:SNAPSHOT-PR-2245-2 fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2245-fabric8-ui.badger.fabric8.io

**Comment by mindreeper2420** [comment]
@dlabrecq After a 'light bulb' moment, I realized the issue was with the fixed positioning of the 'Add Codebase' action when below 1440 screens. I removed that media query, set positioning of the tooltip to "top" (matching the Filter tooltip), and now it works pretty well.

## **1440 + width**
screen shot 2017-11-20 at 5 42 49 pm

## **990 width**
screen shot 2017-11-20 at 5 42 41 pm

## **760 and below width**
screen shot 2017-11-20 at 5 42 34 pm

**Comment by fabric8cd** [comment]
@mindreeper2420 fabric8/fabric8-ui:SNAPSHOT-PR-2245-3 fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2245-fabric8-ui.badger.fabric8.io

**Pull request merged**